code/modules/mob/living/living.dm

```diff
@@ -316,6 +316,10 @@
 /mob/living/is_drawable(allowmobs = TRUE)
 	return (allowmobs && can_inject())
 
+#define NO_SWAP 0
+#define SWAPPING 1
+#define PHASING 2
+
```

Comment:
Who coded this and can they explain these changes to me

Reply:
NO_SWAP means you bump. That's what happening when you in harm intent and trying to pass another xeno
SWAPPING is what happen when you are in help intent and moving into another regular xeno
PHASING is a new interaction, that only happen with spiderling rn. It means that the bumped xeno stays on its tile, while the bumping xeno goes over it.
That's needed (I think) so that widow's movement (and other xenos') are not impeded by the mass of spiderlings